week-1-practise/level-3/1-piping.js

```diff
@@ -11,7 +11,10 @@
 */
 
 var startingValue = 5;
-
+function piping(value) {
```

Comment:
Not quite, we want to use the `add`, `multiply` and `format `  functions below

Reply:
Yes. I noticed when I had sent the pull request. I have fixed my codes.

let result = format(add(multiply(startingValue, 2), 10));